app/controllers/assets_controller.rb

```diff
@@ -20,6 +20,7 @@ def file_present
           # If check permission passes, return :ok
           render json: {
             'preview-url' => @asset.url(:medium),
```

Comment:
Since you used 'get-preview-url', I would then rename 'preview-url' to 'image-tag-url'. Also, it's better to rename 'get-preview-url' just to  'preview-url'.